modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,20 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Advanced Templates
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Templates** premium plugin.
+
+**Advanced Templates** 1.2.0 includes one addition:
+
+=== New `advtemplate_templates` configuration option to define an immutable list of predefined templates.
+//#TINY-9755
+
+In {productname} 6.5, the New `advtemplate_templates` configuration option was introduced. This feature allows users to define an unchangeable list of predefined templates. It is designed for situations where the template list should remain static. Instead of setting up a persistent template store and dealing with a lengthy plugin configuration, users can simply specify a static list of templates that can be easily inserted using the Templates dialog. This option simplifies the process for scenarios where a fixed set of templates is required.
+
+[NOTE]
+Read more about setup configuration for the `advtemplate_template` option xref:ROOT:advtemplate_templates.adoc[here]
```

Comment:
```suggestion
=== Advanced Templates

The {productname} 6.5 release includes an accompanying release of the **Advanced Templates** premium plugin.

**Advanced Templates** 1.2.0 includes one addition:

=== New `advtemplate_templates` configuration option to define an immutable list of predefined templates.
//#TINY-9755

**Advanced Templates** 1.2.0 adds the new `advtemplate_templates` configuration option.

This option can set up an unchangeable list of predefined templates. It is designed for situations where the template list should remain static.

Instead of setting up a persistent template store and the consequent complex plugin configuration, the `advtemplate_templates` option specifies a static templates list that can be inserted using the Templates dialog.

For information on the **Advanced Templates** plugin see: xref:advtemplate_templates.adoc[Advanced Templates].
```

Edits.